### DISCUSSION TITLE
squelch sensitivity

### DISCUSSION THREAD

**Discussion by Costi84** [egzumer/uv-k5-firmware-custom#159] [Ideas]
squelch sensitivity is to hi compared to other firmware , I would  liked it to be set lower, to be more sensitive in step 1

**Comment by ErikS-web** [comment]
Can you clarify the situation?
for example:
- What other firmware
- Did you test with latest 0.19 ?
- What frequency
- What modulation
- How strong was the opposing station

**Reply by Costi84** [reply]
- I compared with facci-mod-v1.2.X-special-converter and Matoz_V0.31.3C
- Yes, I tested with the last 0.19
- I tested in 439.100 mhz fm modulation
- the signal of the station I was listening to was quite weak, that's what I wanted to test, and with 0.19 I had to open the squelch manually, otherwise I couldn't hear anything, and with the other 2 firmwares it opened very well

**Reply by iz8aju** [reply]
Good morning, I also encountered a difficulty with the squelch, as if an intermediate position between 0 and 1 was missing.
My observation occurred with another radio, Yaesu VX3.
Firmware 0.19, mode FM, Mhz 145.525 and 431.225.
Best regards.

**Reply by kamilsss655** [reply]
Same here. It would be great to have intermediate steps between 0 and 1 squelch. :)

**Reply by Flo250578** [reply]
Hello and congratulations!  It is by far the most successful and complete firmware.  But still I found 2 small problems and I can live with them.  But to be the perfect firmware, I will expose the 2 problems here.  
- The first is as my previous colleague also said, I noticed that sometimes the squelch does not open or is unstable when the signal is weak.  I solve this by pressing the monitor button.  but it's not normal like that.  I tested in 70cm and 11m.  
- The second problem is that in ssb the bandwidth change from 25k, to 12.5k to 6.25 is not translated into the sound.  
Otherwise it is ok and has functions that you cannot find in other software.(Firmware 0.19)

**Reply by slavikby** [reply]
Unfortunately, I couldn’t achieve an open squelch in position “1” by changing the variables for the first and second ranges. Also, unfortunately, it was not possible to lower the minimum squelch threshold... Perhaps there are some differences in the calibrations of different versions of main PCB.

**Reply by egzumer** [reply]
If it opens on sql 0 then it will open for any other if set correctly. Either you choose wrong sql index, or wrong band, or you didn't update the driver. Squelch numbering in chirp should start from 0 now and you should be modifing sql1.
I tested with sql 4, band 1-3, frequency 144.2MHz and it worked.

**Reply by slavikby** [reply]
OK. For each squelch level value in your program, I see 6 different constants. What combination of them in your case led to the opening of the squelch?

**Reply by egzumer** [reply]
I told you, Rssi 0, noise 127, glitch 255
The same values for open and close if that is not obvious.

**Reply by slavikby** [reply]
I tried EXACTLY THIS way again, changing the values ​​for SQL1 for both ranges. Squelch won't open... I'm using your latest CHIRP and your latest station driver.

**Reply by egzumer** [reply]
Does it show the newly entered values when you read the configuration back from rafio? Did you enable the "Upload calibration" as the istruction says?

**Reply by slavikby** [reply]
Yes, it looks like the calibration is downloaded to the station, but is not saved there. I'm using your latest version of Chirp.

**Reply by egzumer** [reply]
Then you didn't enable upload calibration option. Just checked it works.

**Reply by slavikby** [reply]
I read your instructions, but unfortunately I didn’t see it or missed it. Please tell me - in which menu item is the required option enabled?

**Reply by slavikby** [reply]
That's it, thank you, I found everything on my own!!!! Thanks for the tip, sorry for taking up time!!!!!

**Reply by egzumer** [reply]
In the Calibration

**Reply by slavikby** [reply]
Thank you, I’m ashamed that I was inattentive!

**Comment by AndrewTXJ** [comment]
Same problem here

**Comment by egzumer** [comment]
you can adjust the sql calibration: https://github.com/egzumer/uvk5-chirp-driver/releases/tag/v0.8

**Reply by slavikby** [reply]
Hello. I tested this option for adjusting the squelch levels, but unfortunately, during the tests I did not find any effect on the threshold...

**Reply by egzumer** [reply]
If that doesn't work then there is no other way of adjusting sql, but I would rather say that you did something wrong.

**Reply by egzumer** [reply]
ok there is an issue, indexes are shifted by one. SQL 1 in the radio is SQL 2 in the chirp and so on

**Reply by egzumer** [reply]
and bands are switched

**Reply by egzumer** [reply]
I published fixed driver.

**Reply by slavikby** [reply]
Thank you very much for your help and your efforts! 
If possible, and if you know -  approximately what range of variable values ​​will be for a fully open squelch? 
Thank you !

**Reply by egzumer** [reply]
Rssi 0, noise 127, glitch 255

BTW, sql 0 from calibration is not used by the radio, it is forced to those values.

**Reply by dadanielat** [reply]
What is 'frequency band' 1-3 and 4-7 referring to?
